src/server.c

```diff
@@ -3628,6 +3640,11 @@ void call(client *c, int flags) {
     /* Do some maintenance job and cleanup */
     afterCommand(c);
 
+    /* Remember the replication offset of the client, right after it's last
```

Comment:
`<nitpick>`

```suggestion
    /* Remember the replication offset of the client, right after its last
```

`</nitpick>`